### PR TITLE
[18.0][MIG] mgmtsystem_manual: Migration to 18.0

### PR DESCRIPTION
**Pull request by natuan9**
Depend on:
- [x] https://github.com/OCA/knowledge/pull/504

### CODE REVIEW

**Review by ajaniszewska-dev — APPROVED** [review]

**Comment by henrybackman** [comment]
When creating a new manual, the form view looks broken:
mgmt manual

**Review by henrybackman — COMMENTED** [review]
Didn't spot anything in the code review but when testing on runboat the new manual form looks wrong

**Review by max3903 — APPROVED** [review]
Form looks good. Just need to squash the migration commits to merge.

File: mgmtsystem_manual/views/mgmtsystem_manual.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<odoo>
+    <record model="ir.actions.act_window" id="action_manuals">
+        <field name="name">Manuals</field>
+        <field name="res_model">document.page</field>
+        <field name="view_mode">list,form</field>
+        <field name="view_id" ref="document_page.view_wiki_tree" />
+        <field name="search_view_id" ref="document_page.view_wiki_filter" />
+        <field name="help">Manuals of your management systems.</field>
+        <field name="domain" eval="[('parent_id','=',ref('manuals'))]" />
+        <field name="context" eval="{'default_parent_id': ref('manuals')}" />
```
Comment (max3903) [inline mgmtsystem_manual/views/mgmtsystem_manual.xml]:
```suggestion
        <field name="domain" eval="[('parent_id', 'ilike', 'manual')]" />
```

File: mgmtsystem_manual/static/description/icon.png
Comment (max3903) [inline mgmtsystem_manual/static/description/icon.png:1]: @natuan9 Please update the icon using https://github.com/OCA/management-system/tree/18.0/mgmtsystem/static/description

**Comment by natuan9** [comment]
Hi @max3903 , I updated domain and icon as your request

**Comment by natuan9** [comment]
> Didn't spot anything in the code review but when testing on runboat the new manual form looks wrong

Hi @henrybackman , thanks for your review, I attempted to replicate the issue you raise, but the new manual form is working fine on my end. I suspect there might be an issue with `document_page` module. Could you please try to fetch the latest changes from the `OCA/knowledge` repository?

image

**Comment by henrybackman** [comment]
> > Didn't spot anything in the code review but when testing on runboat the new manual form looks wrong
> 
> Hi @henrybackman , thanks for your review, I attempted to replicate the issue you raise, but the new manual form is working fine on my end. I suspect there might be an issue with `document_page` module. Could you please try to fetch the latest changes from the `OCA/knowledge` repository?
> 

Works now for me too :+1:

File: mgmtsystem_manual/checklog-odoo.cfg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[checklog-odoo]
+ignore=
+WARNING.0 failed, 0 error\(s\).
```
Comment (dreispt) [inline mgmtsystem_manual/checklog-odoo.cfg]: I'm curious, I'm not aware of this meta file being needed. Was it added by mistake?

**Comment by max3903** [comment]
@natuan9 Can you squash the last 3 commits?

File: mgmtsystem_manual/checklog-odoo.cfg (hunk 2 above)
Comment (natuan9) [reply mgmtsystem_manual/checklog-odoo.cfg]: `checklog-odoo.cfg` is added to avoid test errors

**Pull request merged**